lib/remote/jsonrpcconnection-pki.cpp

```diff
@@ -289,6 +266,9 @@ void JsonRpcConnection::SendCertificateRequest(const JsonRpcConnection::Ptr& acl
 
 	/* Path is empty if this is our own request. */
 	if (path.IsEmpty()) {
+		Log(LogInformation, "JsonRpcConnection")
+			<< "Requesting new certificate for this Icinga instance.";
```

Comment:
This could still log the endpoint name if `aclient != nullptr`. Maybe also add something in the other case at the end, the shortest I can quickly come up with would be "from all connected non-children" (better suggestions for something short saying "nodes in the local and parent zone" are welcome).